TicTacToe.Test/ValidationTest.cs

```diff
@@ -6,11 +6,22 @@ namespace TicTacToe.Test
     public class ValidationTest
     {
         [Fact]
-        public void WrongPlayerSymbol()
+        public void WrongPlayerSymbolAlphabet()
         {
             Validation validate = new Validation();
             char symbol = 'a';
             bool actual = validate.PlayerSymbol(symbol);
+            Assert.Equal("Symbol must be x and o", validate.ErrorMessage);
+            Assert.False(actual);
+        }
+
+        [Fact]
+        public void WrongPlayerSymbolInteger()
+        {
+            Validation validate = new Validation();
+            char symbol = 'a';
+            bool actual = validate.PlayerSymbol(symbol);
+            Assert.Equal("Symbol must be x and o", validate.ErrorMessage);
```

Comment:
test name suggests you testing integer as symbol, yet the `symbol` on `line 22` is a `char`